src/QuadForm/Quad/ZLattices.jl

```diff
@@ -564,6 +564,41 @@ function _root_lattice_E(n::Int)
   return matrix(ZZ, G)
 end
 
+################################################################################
+#
+#  Hyperbolic plane
+#
+################################################################################
+
+@doc Markdown.doc"""
+    Zlattice(R::Symbol, n::Union{Int64, fmpz} = 1) -> Zlat
+
+Given `R = :H` or `R = :U`, return the hyperbolic plane with intersection form
+scaled by `n`.
```

Comment:
```suggestion
scaled by `n`, that is, with scale `n`.
```
I had to look up what intersection form" is :)

Reply:
Yes sorry, since I called it "hyperbolic plane", I used intersection form instead of gram matrix, mainly because I didn't know how to name this matrix, neither how to print it. That's also why I put examples in the docstring to make it clear. :)